plugin/cache.go

```diff
@@ -177,6 +177,8 @@ func CachedList(ctx context.Context, p Parent) (map[string]Entry, error) {
 			// where the context doesn't include the parent's ID.
 			id := strings.TrimRight(p.id(), "/") + "/" + cname
 			entry.setID(id)
+
+			passAlongWrappedTypes(p, entry)
```

Comment:
I'm going to add unit tests for the EntrySchema stuff once the metadata schema work is finished, so am holding off on updating CachedList's unit tests until that point.

Note that I did test this with curl + `/fs/schema?path=$(realpath mnt)` -- the wrapped types are working OK.